user_register_confirm.php

```diff
@@ -0,0 +1,34 @@
+<?php
+require "header.php";
+$user_name = $_GET['user_name'];
+$password = $_GET['password'];
+$password_confirm = $_GET['password_confirm'];
+$admin = $_GET['admin'];
+
+if($admin != 'on'){
+    $admin == 'off';
```

Comment:
==にしてしまうと代入でなく判定になってしまうので
```suggestion
    $admin = 'off';
```
にした方が良さそう。